### DISCUSSION TITLE
FDS parallel computing runs well on my windows machine, but won't run on Linux system

### DISCUSSION THREAD

**Discussion by Yunlongjasonliu** [firemodels/fds#13618] [General]
I got this error message:
[](url)
![FDSParallel](https://github.com/user-attachments/assets/0920156d-a535-438f-bd78-0a6676277e54)

**Comment by Yunlongjasonliu** [comment]
My total mesh count is 3.1 million cells, and 5 mesh blocks.

**Comment by marcosvanella** [comment]
You are probably overrunning your stack memory. Add:
ulimit -s unlimited 
In your startup file. Don't forget to log out and back in to update your environment.

**Comment by mcgratta** [comment]
If this does not help, post your input file and we will try to run it on our computer, and we will see how much memory it requires.

**Comment by Yunlongjasonliu** [comment]
The problem is resolved after I applied these two commands that FDS users Guide recommended. Thanks Kevin.
